<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZExportDialog</class>
 <widget class="QDialog" name="ZExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export pages</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Export pages to single files in designated directory.
Starting from current displayed page.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Total page count</string>
       </property>
       <property name="buddy">
        <cstring>pageCount</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pageCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Export to directory</string>
       </property>
       <property name="buddy">
        <cstring>exportDir</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="exportDir">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportDirBtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Files format</string>
       </property>
       <property name="buddy">
        <cstring>imageFormat</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="imageFormat">
       <item>
        <property name="text">
         <string>PNG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JPG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BMP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GIF</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Compression quality</string>
       </property>
       <property name="buddy">
        <cstring>imageQuality</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="imageQuality">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>90</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pageCount</tabstop>
  <tabstop>exportDir</tabstop>
  <tabstop>exportDirBtn</tabstop>
  <tabstop>imageFormat</tabstop>
  <tabstop>imageQuality</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ZExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ZExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
